plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -0,0 +1,458 @@
+        // max custkey => 1500
+        assertUpdate(
+                "INSERT INTO " + tableName + " (custkey, phone, acctbal) " +
+                        "SELECT custkey + 1600, phone, acctbal FROM customer " +
+                        "UNION ALL " +
+                        "SELECT 100000 + custkey, phone, acctbal FROM customer",
+                "SELECT 2 * count(*) FROM customer");
+
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    @Override
+    public void testCreateTable()
+    {
+        String tableName = "test_create_" + randomTableSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + " (a bigint, b double, c varchar) with (primary_key = ARRAY['a'])");
+        assertTrue(getQueryRunner().tableExists(getSession(), tableName));
+        assertTableColumnNames(tableName, "a", "b", "c");
+
+        assertUpdate("DROP TABLE " + tableName);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+
+        assertQueryFails("CREATE TABLE " + tableName + " (a bad_type) WITH (primary_key = ARRAY['a'])", ".* Unknown type 'bad_type' for column 'a'");
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+
+        tableName = "test_cr_tab_not_exists_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (a bigint, b varchar, c double) WITH (primary_key = ARRAY['b'])");
+        assertTrue(getQueryRunner().tableExists(getSession(), tableName));
+        assertTableColumnNames(tableName, "a", "b", "c");
+
+        assertUpdate("CREATE TABLE IF NOT EXISTS " + tableName + " (d bigint, e varchar) WITH (primary_key = ARRAY['e'])");
+        assertTrue(getQueryRunner().tableExists(getSession(), tableName));
+        assertTableColumnNames(tableName, "a", "b", "c");
+
+        assertUpdate("DROP TABLE " + tableName);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+
+        // Test CREATE TABLE LIKE
+        tableName = "test_create_original_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (a bigint, b double, c varchar) WITH (primary_key = ARRAY['a'])");
+        assertTrue(getQueryRunner().tableExists(getSession(), tableName));
+        assertTableColumnNames(tableName, "a", "b", "c");
+
+        String tableNameLike = "test_create_like_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableNameLike + " (LIKE " + tableName + ", d bigint, e varchar) WITH (primary_key = ARRAY['d'])");
+        assertTrue(getQueryRunner().tableExists(getSession(), tableNameLike));
+        assertTableColumnNames(tableNameLike, "a", "b", "c", "d", "e");
+
+        assertUpdate("DROP TABLE " + tableName);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+
+        assertUpdate("DROP TABLE " + tableNameLike);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableNameLike));
+
+        String tableWithAllProperties = "test_create_with_all_properties";
```

Comment:
Is this method overridden for this test-case? If so extract this case to a new test method instead. Try to keep overrides to the minimum required since they make finding differences very difficult and keeping tests in-sync very difficult.